src/test/rustdoc/mixing-doc-comments-and-attrs.rs

```diff
@@ -0,0 +1,26 @@
+#![crate_name = "foo"]
+
+// @has 'foo/struct.S1.html'
+// @count - '//details[@class="rustdoc-toggle top-doc"]/div[@class="docblock"]/p' \
+//     1
+// @has - '//details[@class="rustdoc-toggle top-doc"]/div[@class="docblock"]/p[1]' \
+//     'Hello world! Goodbye! Hello again!'
+
+#[doc = "Hello world!\n\n"]
```

Comment:
FWIW, I find it surprising that this behaves differently from `S2`. But the point of this test is to capture the status quo.

Reply:
Now that you say it, I didn't pay attention to the fact that the `\n` were discarded. How strange... Still a good thing to have a test for it in any case. Can you open an issue about so we can maybe decide to fix it please?

Reply:
Done: #91201

Reply:
Thanks!